docs/readthedocs/source/doc/Orca/Tutorial/yarn.md

```diff
@@ -43,6 +43,24 @@ The difference between yarn-client mode and yarn-cluster mode is where you run y
 
 For yarn-client, the Spark driver runs in the client process, and the application master is only used for requesting resources from YARN, while for yarn-cluster the Spark driver runs inside an application master process which is managed by YARN in the cluster.
 
+To see the train results, you can look into the driver logs. For **yarn-client** mode, you can directly find the driver logs in the console. For **yarn-cluster** mode, an `application_time_id` will be returned (`application_1668477395550_1045` in the following log) when the application master process is completed.
```

Comment:
remove `To see the train results, you can look into the driver logs.`

new line before `For`